src/app_service/service/wallet_account/async_tasks.nim

```diff
@@ -54,13 +54,25 @@ const fetchDetailsForAddressesTask*: Task = proc(argEncoded: string) {.gcsafe, n
       "details": "",
       "error": ""
     }
+    var jsonReponse = %* {
+      "address": address,
+      "alreadyCreated": false,
+      "path": "",
+      "hasActivity": false
+    }
     try:
-      let response = status_go_accounts.getAddressDetails(arg.chainId, address)
+      var response = status_go_accounts.addressExists(address)
+      if response.result.getBool:
+        jsonReponse["alreadyCreated"] = %*true
+      else:
+        response = status_go_accounts.getAddressDetails(arg.chainId, address)
+        jsonReponse = response.result
       sleep(250)
```

Comment:
To simulate fetching details. When user goes to custom derivation section, then set the derivation path which ends with `/` and then open suggested addresses there is a list of 100 of them, then he is able to scan for activity, so this sleep simulate fetching, otherwise looks odd (we can decrease it to 100-150ms, but not less).